.travis.yml

```diff
@@ -2,14 +2,26 @@ sudo: false
 
 language: python
 
-python:
-  - 2.7
-  - 3.4
-  - 3.5
-  - 3.6
-  - nightly
-
 matrix:
+    include:
+        - python: 2.7
+          dist: trusty
+          sudo: false
+        - python: 3.4
+          dist: trusty
+          sudo: false
+        - python: 3.5
+          dist: trusty
+          sudo: false
+        - python: 3.6
+          dist: trusty
+          sudo: false
+        - python: 3.7
+          dist: xenial
+          sudo: true
+        - python: nightly
+          dist: trusty
+          sudo: false
```

Comment:
Indents should be done with 2 spaces, not 4. Please correct the indentation as Travis cannot parse mixed indentation.